draft-seemann-quic-address-discovery.md

```diff
@@ -137,7 +137,7 @@ Port:
 
 : The port number, in network byte order.
 
-This frame MUST only appear in the handshake and in the application data packet
+This frame MUST only appear in the application data packet
```

Comment:
why not add a note about the 0.5RTT and coalesced packet?  Would that not be helpful guidance for implementers?